evap/student/templates/student_index.html

```diff
@@ -120,3 +120,26 @@
         {% endif %}
     {% endfor %}
 {% endblock %}
+
+
+{% block additional_javascript %}
+    {% if vote_finished_course_id >= 0 %}
+        {{ vote_finished_course_id }}
+        <script type="text/javascript">
+            // delete local storage when a questionnaire was successfully saved
+            // find all entries with voting form prefix
+            var prefix = this.location.hostname + '/student/vote/' + {{ vote_finished_course_id }} + 'student-vote-form';
+            var entries = [];
+            for (var i = 0; i < localStorage.length; i++){
+                if (localStorage.key(i).substring(0,prefix.length) == prefix) {
+                    entries.push(localStorage.key(i));
+                }
+            }
+            // remove the items by key
+            for (var i = 0; i < entries.length; i++) {
+                localStorage.removeItem(entries[i]);
+            }
+        </script>
```

Comment:
sysiphus has manuallyReleaseData which should do all this, only that it is probably not possible to directly use this as the form doesn't exist anymore. could you try sending the form via ajax and deleting everything on success? this [http://stackoverflow.com/questions/11534690/how-to-do-a-jquery-callback-after-form-submit](SO question) (the second answer looks better imo) suggests that it should be fairly simple to do.